apps/utils/models.py

```diff
@@ -15,21 +15,28 @@ def compare_with_model(self, new: Self, exclude_fields: list[str], early_abort=F
                 # These are reverse relations, so let's ignore them
                 continue
 
-            current_values = getattr(self, field.attname)
-            new_values = getattr(new, field.attname)
             if field.many_to_many or field.one_to_many:
+                current_values = getattr(self, field.attname)
+                if new:
+                    new_values = getattr(new, field.attname) if new else None
```

Comment:
if statement above already checks for falsy `new`
```suggestion
                    new_values = getattr(new, field.attname)
```